Makefile.config.in

```diff
@@ -52,6 +52,9 @@ MANDIR=@mandir@
 PROGRAMS_MAN_SECTION=@programs_man_section@
 LIBRARIES_MAN_SECTION=@libraries_man_section@
 
+### Where to install documentation
+DOCDIR=@docdir@
+
```

Comment:
Can this go in `Makefile.build_config.in`, please - the aim is not to add new things to the (installed) `Makefile.config` which don't need to be there.

Reply:
Thanks, done.